README.md

```diff
@@ -31,10 +31,10 @@ yarn add @topsort/sdk --save
 
 ### Auctions
 
-To create an auction, use the `createAuction` function. Example:
+To create an auction, first initialize Topsort Client, then use the `createAuction` function. Example:
 
 ```js
-import { TopsortAuction, createAuction } from "@topsort/sdk";
+import { TopsortClient, TopsortAuction } from "@topsort/sdk";
```

Comment:
Can't help but feel there's a lot of "Topsort" everywhere. Do we really need it? We're already in `@Topsort/sdk`.